src/geo/projection/winkelTripel.js

```diff
@@ -19,18 +11,49 @@ export default {
         lng = lng / 180 * Math.PI;
         const phi1 = Math.acos(2 / Math.PI);
         const alpha = Math.acos(Math.cos(lat) * Math.cos(lng / 2));
-        const x = 0.5 * (lng * Math.cos(phi1) + (2 * Math.cos(lat) * Math.sin(lng / 2)) / sinc(alpha)) || 0;
-        const y = 0.5 * (lat + Math.sin(lat) / sinc(alpha)) || 0;
+        const x = 0.5 * (lng * Math.cos(phi1) + (2 * Math.cos(lat) * Math.sin(lng / 2)) / (Math.sin(alpha) / alpha)) || 0;
+        const y = 0.5 * (lat + Math.sin(lat) / (Math.sin(alpha) / alpha)) || 0;
         return {
-            x: s(x),
-            y: 1 - s(y)
+            x: (x / Math.PI + 0.5) * 0.5,
+            y: 1 - (y / Math.PI + 0.5) * 0.5
         };
     },
 
     unproject(x: number, y: number) {
-        // temporary — replace with a proper implementation
-        return new LngLat(
-            (x - 0.5) * 360,
-            (0.5 - y) * 90);
+        // based on https://github.com/d3/d3-geo-projection, MIT-licensed
+        x = (2 * x - 0.5) * Math.PI;
+        y = (2 * (1 - y) - 0.5) * Math.PI;
+        let lambda = x;
+        let phi = y;
+        let i = 25;
+        const epsilon = 1e-6;
+        let dlambda, dphi;
+        do {
+            const cosphi = Math.cos(phi),
+                sinphi = Math.sin(phi),
+                sinphi2 = Math.sin(2 * phi),
+                sin2phi = sinphi * sinphi,
+                cos2phi = cosphi * cosphi,
+                sinlambda = Math.sin(lambda),
```

Comment:
By the double angle formula (with angles cut in half), I believe `sinlambda` is equal (without sign ambiguity) to `2.0 * sinlambda2 * coslambda2`.

![download](https://user-images.githubusercontent.com/572717/128743215-8de1db38-7ac1-4822-bad4-dbae69a11433.png)

Reply:
Good call! Same for `sin(2 * phi)`. Added a commit.